### PR TITLE
Clarify integer count instrument units

### PR DESCRIPTION
**Pull request by pichlermarc**
This PR clarifies the units that should be used for instruments that measure an integer count. 

The spec mentioned that

> Instruments that measure an integer count of something SHOULD use the default unit 1 (the unity) and annotations with curly braces to give additional meaning. For example {packets}, {errors}, {faults}, etc.

Which, due to the use of "and" in the language, could be interpreted that units should be written as `1{packets}`, `1{errors}` and `1{faults}`. According to [UCUM](https://ucum.org/ucum.html), however, `{packets}` semantically is equivalent to `1{packets}`.

To prevent this interpretation, this PR clarifies that the leading `1` should be omitted.

### CODE REVIEW

**Review by jmacd — APPROVED** [review]

**Review by MrAlias — APPROVED** [review]

File: specification/metrics/semantic_conventions/README.md [hunk 1]
```diff
@@ -148,8 +148,7 @@ more clarification in
 
 - Instruments for **utilization** metrics (that measure the fraction out of a
 total) are dimensionless and SHOULD use the default unit `1` (the unity).
-- Instruments that measure an integer count of something SHOULD use the
-default unit `1` (the unity) and
+- Instruments that measure an integer count of something SHOULD only use
 [annotations](https://ucum.org/ucum.html#para-curly) with curly braces to
-give additional meaning. For example `{packets}`, `{errors}`, `{faults}`,
-etc.
+give additional meaning without the leading default unit (`1`). For example
```
Comment (arminru) [inline specification/metrics/semantic_conventions/README.md:153]:
```suggestion
give additional meaning *without* the leading default unit (`1`). For example
```

File: specification/metrics/semantic_conventions/README.md [hunk 2]
```diff
@@ -148,8 +148,7 @@ more clarification in
 
 - Instruments for **utilization** metrics (that measure the fraction out of a
```
Comment (arminru) [inline specification/metrics/semantic_conventions/README.md:149]: CHANGELOG please :)

**Review by arminru — APPROVED** [review]

File: specification/metrics/semantic_conventions/README.md (hunk 2 above)
Comment (arminru) [reply specification/metrics/semantic_conventions/README.md:149]: Also I think the sentence above would read more naturally if you'd change it to the following while you're at it:

> Units should follow the [Unified Code for Units of Measure (UCUM)](http://unitsofmeasure.org/ucum.html)
(need more clarification in
...

**Review by jsuereth — APPROVED** [review]

File: specification/metrics/semantic_conventions/README.md (hunk 1 above)
Comment (pichlermarc) [reply specification/metrics/semantic_conventions/README.md]: Added in 1fee7b8.

File: specification/metrics/semantic_conventions/README.md (hunk 2 above)
Comment (pichlermarc) [reply specification/metrics/semantic_conventions/README.md:150]: Both suggestions added in 1fee7b8 :)

File: specification/metrics/semantic_conventions/README.md [hunk 3]
```diff
@@ -142,14 +142,14 @@ instrument creation, but can be added if there is ambiguity.
...
 total) are dimensionless and SHOULD use the default unit `1` (the unity).
-- Instruments that measure an integer count of something SHOULD use the
-default unit `1` (the unity) and
+- Instruments that measure an integer count of something SHOULD only use
 [annotations](https://ucum.org/ucum.html#para-curly) with curly braces to
-give additional meaning. For example `{packets}`, `{errors}`, `{faults}`,
-etc.
+give additional meaning *without* the leading default unit (`1`). For example
```
Comment (pirgeo) [inline specification/metrics/semantic_conventions/README.md:154]:
```suggestion
give additional meaning *without* the leading default unit (`1`). For example, use
```
would make it even more clear IMO.

**Review by pirgeo — APPROVED** [review]

File: specification/metrics/semantic_conventions/README.md (hunk 3 above)
Comment (pichlermarc) [reply specification/metrics/semantic_conventions/README.md]: Agreed, suggestion added in b48216d. Thanks!

**Comment by bogdandrutu** [comment]
@arminru can you please rebase and merge this PR since you have all the right access?

**Pull request merged**